modules/5_install/install.tf

```diff
@@ -118,6 +118,7 @@ locals {
         proxy_url               = local.proxy.server == "" ? "" : "http://${local.proxy.user_pass}${local.proxy.server}:${local.proxy.port}"
         no_proxy                = var.cidr
         cni_network_provider    = var.cni_network_provider
+        cni_network_mtu         = var.cni_network_provider == "OVNKubernetes" ? var.private_network_mtu - 100 : var.private_network_mtu - 50
```

Comment:
this will most likely need changes when we support a third one.. Can you please add a comment stating that currently only OpenShiftSDN and OVNKubernetes are handled.

Reply:
done